install-linux.sh

```diff
@@ -78,6 +80,28 @@ Please reach out to support@vanta.com for help.
     exit 1
 fi
 
+if [ ! -f "$UUID_PATH" ]; then
+    printf "\033[31m
+Unable to detect hardware UUID – the Vanta Agent is only supported on platforms which provide a value in $UUID_PATH
+\n\033[0m\n"
+    exit 1
+fi
+
+if [ -x "$(command -v fold)" ] && [ -x "$(command -v uniq)" ] && [ -x "$(command -v wc)" ]; then
```

Comment:
do we also want to check for the things in osquery's `kPlaceholderHardwareUUIDList`

Reply:
I'd rather not introduce a list that we can easily add to, since it may encourage us to change this script more frequently than necessary. Instead, I'd rather have this really basic heuristic and prioritize patching osquery so we actually do support these. Though I don't feel strongly – if we do make a list I think I'll just statically add all of the constant UUIDs as well as the ones in that list

Reply:
I think I'd prefer a list since anything in osquery's list  is going to do the ephemeral UUID  nonsense and I  think that looks worse to  customers